northstar/src/runtime/mount.rs

```diff
@@ -263,7 +295,7 @@ async fn setup_and_mount(
 
     if !use_verity {
         let device = loop_device.path().await.unwrap();
-        mount(&device, root, fs_type, MountFlags::MS_RDONLY, None).await?;
+        mount(&device, root, &FS_TYPE, MountFlags::MS_RDONLY, None).await?;
```

Comment:
If we assume to have squashfs in the npk it should be validated in `Npk::new`. On the other hand a npk is not assembled manually and therefore is just contains what is done in sextant. Never mind.